app/services/imports/scheme_import_service.rb

```diff
@@ -0,0 +1,61 @@
+module Imports
+  class SchemeImportService < ImportService
+    def create_schemes(folder)
+      import_from(folder, :create_scheme)
+    end
+
+    def create_scheme(xml_document)
+      old_id = string_or_nil(xml_document, "id")
+      status = string_or_nil(xml_document, "status")
+
+      return unless status == "Approved"
```

Comment:
Worth logging scheme files that were processed but not imported due to status?

Reply:
Will do now